src/main/java/nl/tudelft/cse1110/andy/config/MetaTest.java

```diff
@@ -1,140 +1,44 @@
 package nl.tudelft.cse1110.andy.config;
 
-import java.util.ArrayList;
-import java.util.List;
-import java.util.stream.Collectors;
+import nl.tudelft.cse1110.andy.execution.externalprocess.ExternalProcess;
+import nl.tudelft.cse1110.andy.execution.metatest.AbstractMetaTestFactory;
 
-public class MetaTest {
+public interface MetaTest {
+    String getName();
 
-    private final int weight;
-    private String name;
-    private MetaEvaluator evaluator;
+    int getWeight();
 
-    private MetaTest(int weight, String name, MetaEvaluator evaluator) {
-        this.weight = weight;
-        this.name = name;
-        this.evaluator = evaluator;
-    }
-
-    public String getName() {
-        return this.name;
-    }
-
-    public int getWeight() {
-        return weight;
-    }
-
-    public static MetaTest withStringReplacement(int weight, String name, String old, String replacement) {
-        MetaEvaluator evaluator = new StringReplacementEvaluator(old, replacement);
-        return new MetaTest(weight, name, evaluator);
-    }
+    String getNameAndWeight();
 
-    public static MetaTest withStringReplacement(String name, String old, String replacement) {
-        return withStringReplacement(1, name, old, replacement);
+    static MetaTest withStringReplacement(int weight, String name, String old, String replacement) {
```

Comment:
Is the `default` visibility on purpose?

Reply:
That is in an interface - the visibility is actually public.

Reply:
Oh yeah, of course, I missed that.